onnxruntime/core/providers/cuda/math/binary_elementwise_ops.cc

```diff
@@ -285,10 +285,18 @@ BINARY_OP_VERSIONED_UZILHFD(Sub, 7, 12)
 BINARY_OP_VERSIONED_UZILHFD(Mul, 7, 12)
 BINARY_OP_VERSIONED_UZILHFD(Div, 7, 12)
 
-BINARY_OP_UZILHFD(Add, 13)
-BINARY_OP_UZILHFD(Sub, 13)
-BINARY_OP_UZILHFD(Mul, 13)
-BINARY_OP_UZILHFD(Div, 13)
+BINARY_OP_VERSIONED_UZILHFD(Add, 13, 13)
+BINARY_OP_VERSIONED_UZILHFD(Sub, 13, 13)
+BINARY_OP_VERSIONED_UZILHFD(Mul, 13, 13)
+BINARY_OP_VERSIONED_UZILHFD(Div, 13, 13)
+BINARY_ELEMENTWISE_REGISTER_KERNEL_VERSIONED_TYPED(Add, 13, 13, BFloat16)
+BINARY_ELEMENTWISE_REGISTER_KERNEL_VERSIONED_TYPED(Sub, 13, 13, BFloat16)
```

Comment:
Same question here. Can we have a new macro instead of having to remember to manually add BFloat16?